src/Sarif/Errors.cs

```diff
@@ -411,7 +411,7 @@ public static RuntimeConditions LogUnhandledEngineException(IAnalysisContext con
             string exceptionMessage = exception?.Message;
             if (!string.IsNullOrEmpty(exceptionMessage))
             {
-                message += " ('" + exceptionMessage + "')";
+                message += Properties.Resources.OpenParenthesis + exceptionMessage + Properties.Resources.CloseParenthesis;
```

Comment:
There is already a .resx file src\Sarif\SdkResources.resx. Please place any new strings there.

But I wouldn't create resource strings for the individual punctuation marks. What the code is doing here is formatting an additional element into the message, in the case where there is an exception to report. So I'd define a resource string `Resources.NotificationWithExceptionMessage` with the value `"{0} ({1})"`. Then I'd write:

```
message = string.Format(CultureInfo.InvariantCulture, Resources.NotificationWithExceptionMessage, message, exceptionMessage);
```